docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,189 @@
+```
+
+Leave the script running in your shell. Teleport will run in the foreground. You
+will see a message in the Web UI that your new server connected successfully:
+
+![Connected
+successfully](../../../img/cloud/getting-started/successfully-connected.png)
 
-Install client libraries:
 
-<Tabs>
-  <TabItem label="MacOS">
-(!docs/pages/includes/enterprise/install-macos-ent.mdx!)
-  </TabItem>
-  <TabItem label="Windows">
-(!docs/pages/includes/install-windows.mdx!)
-  </TabItem>
+Choose the OS user you will assume when you authenticate to the server, in this
+case, `root`:
 
-  <TabItem label="Linux">
-    ```code
-    $ curl -O https://cdn.teleport.dev/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    # verify signature 
-    $ echo "$(curl  https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256)" | sha256sum --check
-    $ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    $ cd teleport-ent
-    $ sudo ./install
-    ```
-  </TabItem>
-</Tabs>
+![Choose your user](../../../img/cloud/getting-started/set-up-access.png)
 
-Login into Teleport and test the connection:
+From there, you can test the connection and access the server. Click **Start
+Session** to connect:
+
+![Start session](../../../img/cloud/getting-started/test-connection.png)
+
+<Notice type="tip">
+
+If you get an error connecting to your container, interrupt the `teleport`
+process in your container's shell with **Ctrl-C**, then run `teleport start` to
+restart the process.
+
+</Notice>
+
+You should see a terminal prompt in your browser. Your Teleport Enterprise Cloud
+account proxies your SSH connection through the Teleport Proxy Service, which
+connects to your container via a reverse tunnel. 
+
+## Step 3/4. Access your server from the command line
+
+Log in to your Teleport Enterprise Cloud tenant via the command line, filling in
+the URL of your tenant and the name of your Teleport user:
+
+```code
+$ tsh login --proxy=<Var name="tenant-url" /> --user=<Var name="username" />
+```
+
+List the servers your Teleport user can access. You should see the name of the
+container you just registered:
 
 ```code
-# tsh logs you in and receives short-lived certificates
-$ tsh login --proxy=myinstance.teleport.sh --user=email@example.com
 $ tsh ls
-# Node Name Address    Labels 
-# --------- ---------- ------ 
-# myserver  ⟵ Tunnel     
-$ tsh ssh root@myserver
 ```
 
-<Details title="Not seeing Nodes?" opened={false}>
+Access your server as the `root` user:
+
+```code
+$ tsh ssh root@<Var name="server-name" />
+```
+
+Run some commands in your terminal. In the next step, we will show you how you
+can play back your session. When you have finished, run `exit` to end your
+session.
+
+## Step 4/4. Play back your session
+
+While Teleport Enterprise Cloud proxies SSH connections to registered servers,
+it stores recordings of SSH sessions so you can play them back later to
```

Comment:
Consider:

```suggestion
it records SSH sessions so you can play them back later to
```

In the current form the "While" could be read as the second conjunction form, indicating a contrast. IMHO this indicates the non-contrasting conjunction better.